scripts/kanzus_dev

```diff
@@ -0,0 +1,219 @@
+#!/local/data/idsbc/idstaff/gladier/miniconda3/envs/gladier/bin/python
+
+import pathlib
+import time, argparse, os
+from watchdog.observers import Observer
+#from watchdog.observers.polling import PollingObserver as Observer
+from watchdog.events import FileSystemEventHandler
+
+class KanzusTriggers:
+    def __init__(self, folder_path):
+        self.observer = Observer()
+        self.folder_path = folder_path
+
+    def run(self):
+        print("Kanzus Triggers Started")
+        if not os.path.isdir(self.folder_path):
+            print("Monitor dir does not exist.")
+            print("Dir " + self.folder_path + " was created")
+            os.mkdir(self.folder_path)
+            
+        print("Monitoring: " + self.folder_path)
+        print('')
+
+        event_handler = Handler()
+        self.observer.schedule(event_handler, self.folder_path, recursive = True)
+        
+        self.observer.start()
+        
+        try:
+            while True:
+                time.sleep(1)
+        except:
+            self.observer.stop()
+            print("Kanzus Triggers Stopped")
+
+        self.observer.join()
+
+#https://stackoverflow.com/questions/58484940/process-multiple-oncreated-events-parallelly-in-python-watchdog
+class Handler(FileSystemEventHandler):
+    @staticmethod
+    def on_any_event(event):
+#        print(event)
+        if event.is_directory:
+            return None
+        elif event.event_type == 'created':
+            #KanzusLogic(event.src_path)
+            return None
+        elif event.event_type == 'modified':
+            KanzusLogic(event.src_path)
+            return None
+
+def parse_cbf_event(event_file):
+    event = {}
+    event['exp'] = event_file.split('/')[-4]
+    event['sample'] = event_file.split('/')[-3]
+    event['chip_letter'] = event_file.split('/')[-2]
+    event['filename'] = event_file.split('/')[-1]
+    event['chip_name'] = event['filename'].split('_')[0]
+    event['run_num'] = int(event['filename'].split('_')[1])
+    try:
+        event['cbf_num'] = int(event['filename'].split('_')[2].replace('.cbf',''))
+    except:
+        event['cbf_num'] = None
+    return event
```

Comment:
This `parse_cbf_event` function is a duplicate in folder_watch.py

Reply:
folder_watch is still just a prototype. I left it there to test the filesystem in case they change it.
I will remove the parser.

Do you think I should move those parsers into a different `.py` or leave it on the client?